pyannote/audio/labeling/tasks/speech_activity_detection.py

```diff
@@ -162,14 +162,18 @@ class DomainAwareSpeechActivityDetection(SpeechActivityDetection):
 
     DOMAIN_PT = '{log_dir}/weights/{epoch:04d}.domain.pt'
 
-    def __init__(self, domain='domain', attachment=-1, **kwargs):
+    def __init__(self, domain='domain', attachment=-1, rnn=None, **kwargs):
```

Comment:
Can you please add "rnn" and its description in the docstring?
You can simply offer to have a look at `pyannote.audio.models.models.RNN` docstring...